### DISCUSSION TITLE
Network/Graph view

### DISCUSSION THREAD

**Discussion by maxmar391** [Zettlr/Zettlr#2384] [Ideas]
Dear all,
I have recently learned about Zettlr and use the program on Windows. I like it a lot as note taking software! It has some great features and overall I prefer it slightly over Obsidian. However, there is one feature why I actually use both in parallel: the graph view of Obsidian. I find this a very helpful feature to see how my notes are interlinked. 
I would like to see you implement something similar.
Are there other people that would like to see this feature?
All the best,
Max

**Comment by nathanlesage** [comment]
Related PR: https://github.com/Zettlr/Zettlr/pull/921

**Comment by maxmar391** [comment]
Agreed. It is quite a long thread. I am not sure, what is the status/outcome of PR #921 ? Will this feature be incorporated at all? or when will it be incorporated?

**Reply by nathanlesage** [reply]
I'm right now confident that it will be implemented, but it takes a lot of time. PR 921 is unfortunately already quite outdated and I don't know if the original creator is still around.

**Reply by StoltHD** [reply]
@nathanlesage -when you eventually get the time to consider it again, do it simple, don't add a million controllers to the graph view it self, for every single filter for styling color, shape etc. that can be used.

Let the users set those settings and styles as rules in a JSON file, easily accessible...

Most users that want to use a graph view with extended styling, usually manage to get a round setting up rules in a formatted json file.
Filters for what is displayed in the graph should still be a GUI option, where you can set i.e. folders (paths) not to include in the graph etc.
_Just my tips after using both Obsidian and Foam in my "research" for a while, including the great Juggl addon for Obsidian_.

Users that want to set styles on their graphs will most likely seldom change the styles after the initial configuration, so it is not a problem having them semi-hidden in a json file.
